metadata-store-directory-common/pom.xml

```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.10.2</version>
+      <version>2.11.0</version>
```

Comment:
I'm a bit confused about the lib changes in several pom files. Would you explain a bit? Also does it require any ivy file change later?

Reply:
Sure.
1. jackson-databind depends on jackson-core, so no need to explicitly specify jackson-core
2. If jackson 2.11.0 is used in helix, I believe it is better to make the version consistent. So change 2.10.2 -> 2.11.0
3. Yeah, better to also change the version in Ivy.